### PR TITLE
Merged foodborne training updates

### PR DESCRIPTION
**Pull request by EngyNasr**
Merged Updates from my closed updates in PR [#3852](https://github.com/galaxyproject/training-material/pull/3852) and Wolfgang's updates from PR [#3853](https://github.com/galaxyproject/training-material/pull/3853)

Thanks a lot :)

### CODE REVIEW

**Comment by bebatut** [comment]
@EngyNasr Thanks
Could you also address here @wm75 comments here #3854? Thanks a lot

File: faqs/galaxy/sequencing_nanopore.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Nanopore sequencing has several properties that make it well-suited for our purp
 2. Long-read sequencing gives the ability to **span repetitive genomic regions**
 3. Long-read sequencing makes it possible to **identify large structural variations**
 
-![How nanopore sequencing works]({% link topics/metagenomics/images/plasmid-metagenomics-nanopore/sequence_method.jpg %} "Using nanopore sequencing, a single molecule of DNA or RNA can be sequenced without the need for PCR amplification or chemical labeling of the sample. (Image from: <a href="https://nanoporetech.com/sites/default/files/s3/white-papers/WGS_Assembly_white_paper.pdf?submissionGuid=40a7546b-9e51-42e7-bde9-b5ddef3c3512">Nanopore sequencing: The advantages of long reads for genome assembly</a>)") <br><br>
+![How nanopore sequencing works]({% link topics/metagenomics/images/plasmid-metagenomics-nanopore/nanopore_sequencing.png %} "Using nanopore sequencing, a single molecule of DNA or RNA can be sequenced without the need for PCR amplification or chemical labeling of the sample. (Image from: <a href="https://togotv.dbcls.jp/togopic.2020.01">Nanopore sequencing: The advantages of long reads for genome assembly</a>)") <br><br>
```
Comment (wm75) [inline faqs/galaxy/sequencing_nanopore.md:16]: the new Image from link isn't working for me.

File: topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/workflows/nanopore_preprocessing.ga [hunk 2]
```diff
@@ -50,8 +50,8 @@
             "name": "Input dataset",
             "outputs": [],
             "position": {
-                "left": 0,
-                "top": 600.9666748046875
+                "left": 0.0,
```
Comment (wm75) [inline topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/workflows/nanopore_preprocessing.ga:53]: You can restore this whole file. It only contains non-significant changes now.

File: topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md [hunk 3]
```diff
@@ -252,7 +251,7 @@ In this tutorial we use similar tools as described in the tutorial ["Quality con
 
 > <question-title></question-title>
 >
-> Inspect the webpage output of **MultiQC** for `Barcode10`
+> Inspect the HTML output of **fastp** and **MultiQC** for `Barcode10`
```
Comment (wm75) [inline topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md:254]: Rebase artefact - remove.

File: topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md [hunk 4]
```diff
@@ -263,6 +262,8 @@ In this tutorial we use similar tools as described in the tutorial ["Quality con
 > > 1. Before trimming the file has 114,986 sequences and After trimming the file has 91,434 sequences
 > > 2. The "Per base sequence quality" is globally medium: the quality score stays above 20 over the entire length of reads after trimming, while quality below 20 could be seen before trimming specially at the beginning and the end of the reads.
 > >
+> >    ![fastp tool report of Barcode10 sample before and after retaining the avarage quality of the reads](./images/fastp_report_barcode10.png)
```
Comment (wm75) [inline topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md:265]: same thing

File: topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md [hunk 5]
```diff
@@ -1198,7 +1199,7 @@ We use **Heatmap w ggplot** tool along with other tabular manipulating tools to
...
+> > 1. A lot of bacteria pathogen **VF** gene products identified by the **VFDB** are common in both samples, three of them are with the following accession number: **NP_461810**, **NP_461809** and **NP_459541**
 > > 2. **AAG03023** is only found in `Barcode10` sample and **NP_460360** is only found in `Barcode11` sample
 > > 3. Both samples are spiked with the same pathogen species, _S. enterica_, but not the same strain. `Barcode10_Spike2` sample is spiked with _S. enterica subsp. enterica_ strain and `Barcode11_Spike2b` with _S. enterica subsp. houtenae_. This can be the main cause of the most similarities and the few difference found bacteria pathogen genes between both of the samples. Other factors such as the time and location of the sampling may cause other differences. By knowing the metadata of the samples inputted for the sub-workflows in real life we can understand what actually happened. We can have samples with no pathogen found then we start detecting genes from the 7th or 8th sample, then we can identify where and when the pathogen entered the host, and stop the cause of that
```
Comment (wm75) [inline topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md:1204]: Number 3 is the one I'm really struggling with.

File: topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md [hunk 6]
```diff
@@ -33,9 +33,9 @@ contributions:
 
 # Introduction
 
-Food contamination with pathogens are a major burden on our society. In the year 2019, foodborne pathogens caused 137 diseases in Germany [(BVL 2019)](https://www.bvl.bund.de/SharedDocs/Berichte/10_BELA_lebensmittelbed_Krankheitsausbruechen_Dtl/Jahresbericht2019.pdf?__blob=publicationFile&v=4). Globally, they affect an estimated 600 million people a year and impact socioeconomic development at different levels. These outbreaks are mainly due to _Salmonella spp._ followed by _Campylobacter spp._ and Noroviruses.
+Food contamination with pathogens are a major burden on our society. In the year 2019, foodborne pathogens caused 137 diseases in Germany [(BVL 2019)](https://www.bvl.bund.de/SharedDocs/Berichte/10_BELA_lebensmittelbed_Krankheitsausbruechen_Dtl/Jahresbericht2019.pdf?__blob=publicationFile&v=4). Globally, they affect an estimated 600 million people a year and impact socioeconomic development at different levels. These outbreaks are mainly due to _Salmonella spp._ followed by _Campylobacter spp._ and Noroviruses, as studied by the [__Food safety - World Health Organization (WHO)__](https://www.who.int/news-room/fact-sheets/detail/food-safety) in 2022.
```
Comment (wm75) [inline topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md:36]: Couple of issues with this paragraph:
- The WHO link leads to a page that was *updated* in 2022, but if you read the text, the number 600 million is from a *2015* report. That report (https://www.who.int/publications/i/item/9789241565165) makes the better link I suppose.
- It's possible that the pathogen importance order you're giving is contained somewhere in that long report, but it's not given on the page you're currently linking to
- the German (BVL2019) report is also not summarized correctly: 137 is the number of hospitalisations (not diseases) caused by disease outbreaks for which there was strong evidence that they were caused by a foodborne pathogen.

File: faqs/galaxy/sequencing_nanopore.md (hunk 1 above)
Comment (EngyNasr) [reply faqs/galaxy/sequencing_nanopore.md:16]: https://togotv.dbcls.jp/togopic.2020.01.html

Comment (EngyNasr) [reply faqs/galaxy/sequencing_nanopore.md:16]: I am solving these comments right now :)

File: topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/workflows/nanopore_preprocessing.ga (hunk 2 above)
Comment (EngyNasr) [reply topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/workflows/nanopore_preprocessing.ga:53]: I will re download the workflow and put it

Comment (wm75) [reply topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/workflows/nanopore_preprocessing.ga:53]: You just need to git restore the file :-)

File: topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md (hunk 3 above)
Comment (EngyNasr) [reply topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md:254]: done

File: topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md (hunk 4 above)
Comment (EngyNasr) [reply topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md:265]: done

File: faqs/galaxy/sequencing_nanopore.md (hunk 1 above)
Comment (EngyNasr) [reply faqs/galaxy/sequencing_nanopore.md:16]: done

File: topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md (hunk 5 above)
Comment (EngyNasr) [reply topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md:1204]: I guess this time will be better :)

**Review by bebatut — APPROVED** [review]

**Pull request merged**